app/services/parser_linter.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class ParserLinter
+  attr_reader :parser, :tempfile, :warnings
+
+  def initialize(parser)
+    @parser = parser
+    @tempfile = "tmp/parser_script-#{@parser.id}.rb"
+  end
+
+  def lint
+    create_tmp_file
+    run_lingter
+    delete_tmp_file
+
+    @warnings
+  end
+
+  private
+    def create_tmp_file
+      file = File.new(@tempfile, 'w')
+      file << @parser.content
+      file.close
+    end
+
+    def run_lingter
+      @warnings = `bundle exec rubocop #{@tempfile} --format simple`
```

Comment:
have you guys thought about also having a cops file to manage the rules?

Reply:
As of now its using the same default `rubocop.yml` from the project. We are waiting for Dan to flush out the feature and probably configure this differently.